### ISSUE TITLE
Staging Review finding: Placement of sort dropdown

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98158]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

This ticket was part of a merged Staging Review with MHV Medications [List Filter](https://github.com/department-of-veterans-affairs/va.gov-team/issues/88958) and [Prescription Information](https://github.com/department-of-veterans-affairs/va.gov-team/issues/88952)

## Product Information
**Team:** MHV Medications
**Product:** Medications
**Feature:** List Filter & Prescription Information

## Findings details
**VA.gov Experience Standard - issue:** [User does not have a path to the content or feature.](https://design.va.gov/about/experience-standards/#user-has-a-path-to-the-content-or-feature)
**VA.gov Experience Standard - category:** [Findability](https://design.va.gov/about/experience-standards/#category-findability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the [/medications/?page=1](https://staging.va.gov/my-health/medications/?page=1) page, the "Show medications in this order" select element is placed between the "Filter list" accordion and the "Showing X - Y of Z" summary message. This came up in [a big Slack discussion about filtering and sorting](https://dsva.slack.com/archives/C8E985R32/p1726630019711259?thread_ts=1726579725.396949&cid=C8E985R32) which I believe you all took into consideration when designing this pattern. I'm logging this issue for the sake of further discussion, along with a video of the problematic behavior I observe.

When a user opens the filter list, chooses their filter criteria, and presses "Apply filter" (and likewise "Reset filter"), keyboard focus is moved to the "Showing X - Y of Z" message. This is good -- it provides users with immediate feedback that their action has completed and gives them a description of what's shown on the page. But by moving focus directly to the status message, the sort element is skipped over. For screen reader users that don't see the visual layout of the page, the sort element may be missed entirely, since there's no reason a user would decide to work backwards just to explore what else might be hiding in there. I've included a video of this experience.

Is this an issue? Maybe? I'll defer to you on the typical behavior of your users, as perhaps most users work their way through the page content and discover that the sort element exists before they decide to apply a filter. But I do think there's a discoverability cost to the current placement.

<details><summary>Link, screenshot or steps to recreate</summary>

https://github.com/user-attachments/assets/bf83e69c-6d38-4c72-92e9-98d19fe598bc

</details>

## Recommended action

I've suggested to other teams that this could be solved by combining filtering and sorting into a single "Filter & Sort" action --- ie. place the sort options inside the filter accordion, and have the same button trigger both the filter and sort action. Whether this is appropriate to your product is up to you!

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by TannerBird98** [comment]
We have decided to not do anything for this based on the feedback we found with user testing. Filters were initially overlooked as well as some errors when using the filters so combining the two would complicate things further.

**Comment by shiragoodman** [comment]
FYSA @briandeconinck
